test/extended/util/docker.go

```diff
@@ -0,0 +1,85 @@
+package util
+
+import (
+	"fmt"
+	dockerClient "github.com/fsouza/go-dockerclient"
+	tutil "github.com/openshift/origin/test/util"
+)
+
+func TagImage(tagee, tagor string) error {
+	client, dcerr := tutil.NewDockerClient()
+	if dcerr != nil {
+		return dcerr
+	}
+	opts := dockerClient.TagImageOptions{
+		Repo:  tagee,
+		Tag:   "latest",
+		Force: true,
+	}
+	terr := client.TagImage(tagor, opts)
+	return terr
+}
+
+func PullImage(name string) error {
+	client, dcerr := tutil.NewDockerClient()
+	if dcerr != nil {
+		return dcerr
+	}
+	opts := dockerClient.PullImageOptions{
+		Repository: name,
+		Tag:        "latest",
+	}
+	perr := client.PullImage(opts, dockerClient.AuthConfiguration{})
+	return perr
+}
+
+type MissingTagError struct {
+	Tags []string
+}
+
+func (mte MissingTagError) Error() string {
+	return fmt.Sprintf("the tag %s passed in was invalid, and not found in the list of images returned from docker", mte.Tags)
+}
+
+func GetImageIDForTags(comps []string) ([]string, error) {
```

Comment:
Yeah, @bparees and I talked a bit about code reuse here, and I believe where we landed was that I had the to talk directly to the fsouza docker client.  Oracle searches on the fsouza docker client API I'm using did not turn anything up.  And some scans on imagestream (which turned up hits in pkg/client and pkg/generate/app) did not reveal anything useful for this test (I saw stuff around the OpenShift Namespace).

Of course, not having looked in the image stream area, perhaps I missed something.  But pending "you should have looked here dummy" pointer ;-), I'm going to claim due diligence has been served, and leave this function as is.